pmda/test/test_rdf.py

```diff
@@ -83,6 +83,15 @@ def test_same_result(sels, n_blocks):
     assert_almost_equal(nrdf.rdf, prdf.rdf)
 
 
+def test_cdf(sels):
+    s1, s2 = sels
+    rdf = InterRDF(s1, s2).run()
+    rdf.get_cdf()
+    cdf = np.cumsum(rdf.count) / rdf.nf
+    assert rdf.cdf[-1] == rdf.count.sum()/rdf.nf
```

Comment:
Is this comparing floats or integers? If these are floats then you need to use `assert_almost_equal`.

Reply:
Fixed.